simple_ai_agents/email_to_calendar_scheduler/.env.example

```
@@ -0,0 +1 @@
+GROQ_API_KEY=<your-groq-api-key-here>
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)